Authorization/Authorization/Presentation/Startup/StartupViewModel.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  StartupViewModel.swift
+//  Authorization
+//
+//  Created by SaeedBashir on 10/23/23.
+//
+
+import Foundation
+import Core
+
+public class StartupViewModel: ObservableObject {
+    let router: AuthorizationRouter
+    private let interactor: AuthInteractorProtocol
+    private let analytics: AuthorizationAnalytics
+    @Published var searchQuery: String?
+    
+    public init(
+        interactor: AuthInteractorProtocol,
+        router: AuthorizationRouter,
+        analytics: AuthorizationAnalytics
+    ) {
+        self.interactor = interactor
+        self.router = router
+        self.analytics = analytics
+    }
+    
+    func trackSigninClicked() {
+        
```

Comment:
@saeedbashir  do we need this empty function?

Reply:
Not at the moment, I wrote this so it will be helpful in the near future while implementing analytics. But we will write it again according to the requirements. I've removed it.